.github/actions/e2e/env-setup/action.yml

```diff
@@ -66,7 +66,7 @@ runs:
       shell: bash
       run : |
         echo "::group::Kill webserver running on port 8084"
-        sudo fuser -k -n tcp 8084
+        sudo fuser -k -n tcp 8084 || true
```

Comment:
According to [the manual](https://man7.org/linux/man-pages/man1/fuser.1.html)
> fuser returns a non-zero return code if none of the specified files is accessed

So we can just ignore if the port is not busy.